### PR TITLE
Add Visual Regression Testing

### PR DESCRIPTION
**Pull request by pauljeter**
## Description
Add visual regression testing to the Core library with Cypress.io and Percy.io

## Motivation and Context

VR Testing will allow the Momentum team to be confident that changes only affect the components that are intended.

## Screenshots:
image

image

image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: core/tools/base.config.js [hunk 1]
```diff
@@ -19,16 +22,16 @@ const baseConfig = {
       'react-native': 'react-native-web',
       '@collab-ui/icons': path.resolve(codePath, '../icons'),
       'images': path.resolve(codePath, 'images'),
-      // '@collab-ui/core': path.resolve(codePath, 'node_modules/@collab-ui/core/css/collab-ui.css'),
+      // '@collab-ui/core': path.resolve(codePath, '../core'),
```
Comment (bfbiggs) [inline core/tools/base.config.js]: Should this line just be removed?

File: core/tools/base.config.js [hunk 2]
```diff
@@ -61,7 +56,75 @@ const baseConfig = {
...
+            {
+              loader: 'sass-loader',
+              options: {
+                outputStyle: 'compressed',
+                sourceMap: true,
+              }
+            }
+          ]
+        }),
+        // use: ExtractTextPlugin.extract([
```
Comment (bfbiggs) [inline core/tools/base.config.js]: Does this need to be added?

Comment (pauljeter) [reply core/tools/base.config.js]: nope, needs to be removed.

File: core/tools/base.config.js (hunk 1 above)
Comment (pauljeter) [reply core/tools/base.config.js]: Done.

File: core/tools/base.config.js (hunk 2 above)
Comment (pauljeter) [reply core/tools/base.config.js]: Nope, this should be removed.

**Pull request merged**